flare/src/commonMain/kotlin/enchant/flare/FirebaseFunctions.kt

```diff
@@ -1,8 +1,31 @@
 package enchant.flare
 
+/**
+* Firebase functions is ...
```

Comment:
Give some more detail... link to Firebase documentation using link syntax: [link](http://somelink.com)